src/main/gen/wgslplugin/language/psi/impl/WGSLVariableIdentDeclImpl.java

```diff
@@ -42,4 +42,9 @@ public PsiElement getNameIdentifier() {
     return WGSLPsiImplUtil.getNameIdentifier(this);
   }
 
+  @Override
+  public PsiElement setName(String newName) {
+    return WGSLPsiImplUtil.setName(this, newName);
+  }
+
```

Comment:
without `setName` override only variable occurrences are renamed (declaration is unchanged), so I assume it should be declared here